packages/monocdk/test/cfn-include/cfn-include.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as cfn_inc from '../../cloudformation-include';
+import * as s3 from '../../aws-s3';
+import { Stack } from '../../lib';
+
+test('cloudformation-include works in monocdk', () => {
```

Comment:
Move this into the new rewriter package.

Just run it at the end of `cfn-include-mapping-rewriter.ts`. It doesn't have to be jest.
It'll remove the need for all the extra jest files in both these packages.

Reply:
But that will not test what we want, if we remove the Jest files from the packages. They are there for a reason 🙂.

In other words, if I add this test to `cfn-include-mapping-rewriter.ts`, it will pass today, even if I comment out all of the logic that changes the mapping file from `cfn-include-mapping-rewriter.ts`.

Reply:
How come? Why does the test work in jest but not during the build script?

Reply:
Because of this:

```ts
// disallow tests (and production code) in aws-cdk-lib from including the @aws-cdk/core module,
// which effectively bans using any @aws-cdk/* module,
// as they all depend on @aws-cdk/core
jest.mock('@aws-cdk/core', () => {
    throw new Error('@aws-cdk/core is not available in aws-cdk-lib tests!')
});
```

https://github.com/aws/aws-cdk/pull/11595/files?file-filters%5B%5D=.js#diff-491ee0e63a08ee8afaf3fa68d208368bed15285bdaceed49b881c782e44b0b05

Reply:
On thinking about it some more, I have no idea how you even see this working 🙂.

Won't writing this sort of test in `cfn-include-mapping-rewriter`mean a cyclic dependency between that package, and `monocdk` / `aws-cdk-lib`?

Reply:
Shouldn't be, if it's modeled as a build tool and the import is relative to the root of the package being built, no?

Reply:
> Shouldn't be, if it's modeled as a build tool and the import is relative to the root of the package being built, no?

Can you show me the code you expect to be written in `cfn-include-mapping-rewriter.ts` (renamed in this latest iteration of `cfninclude-rewriter.ts`)?

Anyway, I think that test will not actually check what you want, because of https://github.com/aws/aws-cdk/pull/11595#discussion_r532743327 .